ljps-frontend/pages/learning-journey/view-skills/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect, useRef } from "react";
+import axios from "axios";
+import { useRouter } from "next/router";
+import Image from "next/image";
+
+export default function ViewSkills() {
+  var router = useRouter();
+  var roleID = router.query["selectedRole"];
+  const toast = useRef()
+
+  const [roleDescription, setRoleDescription] = useState("");
+  const [roleDepartment, setRoleDepartment] = useState("");
+  const [roleName, setRoleName] = useState("");
```

Comment:
It might be better to have one state as an object that contains all the role properties